# foo
import "../self_hosted/error.oomph" as error

export func main():
    # TODO: something like Python's __file__
    let newline = new error::Location(
        "tests/newline_location.oomph", 1, "# foo", "\n"
    )
    newline.print_error("this error is right where a newline is")
